tests/unit/disco/components/TestDiscoPane.js

```diff
@@ -327,12 +319,25 @@ describe(__filename, () => {
   });
 
   describe('componentDidMount', () => {
-    it('sets events', () => {
+    it('does not set events on the server', () => {
```

Comment:
I guess this ties in to your above change but still not clear why this change is needed..

Reply:
because it was the same result before, but very obscure. it